extensions/cornerstone/src/Viewport/Overlays/CustomizableViewportOverlay.tsx

```diff
@@ -210,9 +213,10 @@ function CustomizableViewportOverlay({
             <div key={`${keyPrefix}_${index}`}>
               {item?.condition
                 ? item.condition({
-                    instance: instances ? instances[item?.instanceIndex] : null,
-                    formatters: { formatDate: formatDICOMDate },
-                  })
+                  currentInstance: instances ? instances.currentInstances[item?.instanceIndex] : null,
+                  referenceInstance: instances ? instances.referenceInstances[item?.instanceIndex] : null,
```

Comment:
You need to get displaySet.instance for the reference instance, since the display set itself determines the reference.